python/cuml/cuml/manifold/t_sne.pyx

```diff
@@ -276,6 +276,11 @@ class TSNE(UniversalBase,
     _hyperparam_interop_translator = {
         "n_components": {
             3 : "NotImplemented",
+        },
+        "method": {
+            "exact" : "exact",
+            "barnes_hut" : "fft",  # mitigation for #6617
```

Comment:
I have a _slight_ preference to mark this as NotImplemented to fallback to CPU. Presumably if a user specified they wanted this method to be used they had a reason. Until the bug is fixed in cuml I'd prefer we do what they say rather than assume that `fft` would be a fine alternative.

Reply:
NVM, just learned that the sklearn default is `barnes_hut` - I can see the desire to have the default hyperparameters always work with cuml. Knowing little about these methods, is there any reason a user might prefer `barnes_hut` over `fft`? AFAIK they're both approximations, and sklearn doesn't implement `fft` only `barnes_hut`.

Reply:
I think the safer choice is to fallback to CPU for now. That's really unfortunate, but hopefully we can resolve the underlying root cause and then re-enable it.

Reply:
Implemented in https://github.com/rapidsai/cuml/pull/6637 .